command/commands/team.py

```diff
@@ -8,97 +8,89 @@ class TeamCommand:
-           "* edit GITHUB_TEAM_NAME"\
-           " [--name DISPLAY_NAME]"\
-           " [--platform PLATFORM]\n"\
-           "    * edit properties of specified team\n\n"\
-           "* add GITHUB_TEAM_NAME SLACK_ID\n"\
-           "    * add the specified user to the team\n\n"\
-           "* remove GITHUB_TEAM_NAME SLACK_ID\n"\
-           "    * remove the specified user from the team\n\n"\
-           "* delete GITHUB_TEAM_NAME\n"\
-           "    * permanently delete the specified team\n"
     desc = "for dealing with " + command_name + "s"
 
     def __init__(self):
         """Initialize team command parser."""
         logging.info("Initializing TeamCommand instance")
         self.desc = ""
-        self.parser = argparse.ArgumentParser(prog="team")
+        self.parser = argparse.ArgumentParser(prog="/rocket")
         self.parser.add_argument("team")
-        self.init_subparsers()
+        self.subparser = self.init_subparsers()
+        self.help = self.get_help()
 
     def init_subparsers(self):
         """Initialize subparsers for team command."""
         subparsers = self.parser.add_subparsers(dest="which")
 
         """Parser for list command."""
         parser_list = subparsers.add_parser("list")
-        parser_list.set_defaults(which="list")
+        parser_list.set_defaults(which="list",
+                                 help="outputs the Github team names "
+                                      "and displays names of all teams")
 
         """Parser for view command."""
         parser_view = subparsers.add_parser("view")
-        parser_view.set_defaults(which="view")
+        parser_view.set_defaults(which="view",
+                                 help="view information and members of "
+                                      "a team")
         parser_view.add_argument("team_name", type=str, action='store')
 
-        """Parser for help command."""
-        parser_view = subparsers.add_parser("help")
-        parser_view.set_defaults(which="help")
-
         """Parser for delete command."""
         parser_delete = subparsers.add_parser("delete")
-        parser_delete.set_defaults(which="delete")
+        parser_delete.set_defaults(which="delete",
+                                   help="permanently delete specified "
+                                        "team (admin only)")
         parser_delete.add_argument("team_name", type=str, action='store')
 
         """Parser for create command."""
         parser_create = subparsers.add_parser("create")
-        parser_create.set_defaults(which="create")
-        parser_create.add_argument("team_name", type=str, action='store')
-        parser_create.add_argument("--name", type=str, action='store')
-        parser_create.add_argument("--platform", type=str, action='store')
-        parser_create.add_argument('--channel', action='store_true')
+        parser_create.set_defaults(which="create",
+                                   help="create a new team with the "
```

Comment:
I think `(admin only)` should be moved to the front of the help string, so you can easily see what the permissions are when scanning the list of commands.

Reply:
Also, `admin` should be capitalized.

Reply:
should the first letter after the ending bracket be capitalized?

Reply:
I think so. I'd prefer something like:
```
create: (Admin only) Create a new team with...
```
But I could see arguments for other capitalizations as well 😛

Reply:
nice i guessed correctly. that is what it should look like now